addons/point_of_sale/static/src/js/models.js

```diff
@@ -2076,9 +2076,9 @@ exports.Orderline = Backbone.Model.extend({
         }
         return taxes;
     },
-    _map_tax_fiscal_position: function(tax) {
+    _map_tax_fiscal_position: function(tax, order) {
```

Comment:
shouldn't we set order=false by default?

Reply:
Yeah that true. I force push the branch for this change.